idseq_dag/util/count.py

```diff
@@ -0,0 +1,30 @@
+import idseq_dag.util.command as command
+
+def reads_in_group(file_group):
+    ''' Count reads in a group of matching files '''
+    num_files = len(file_group)
+    reads_in_first_file = reads(file_group[0])
+    return num_files * reads_in_first_file
+
+def reads(local_file_path):
+    ''' Count reads in a local file '''
```

Comment:
I think more documentation in either this function or line2reads will be nice such as expected file format, etc.

Reply:
Also would be nice if this support .gz files

Reply:
ok